projects/plugins/jetpack/modules/stats.php

```diff
@@ -588,6 +685,248 @@ function stats_reports_page( $main_chart_only = false ) {
 	}
 }
 
+/**
+ * Legacy Stats: Print Header Section
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_header_section( $html ) {
+	$header = stats_parse_header_section( $html );
+	if ( $header !== '' ) {
+		echo $header; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+}
+
+/**
+ * Legacy Stats: Print Content Section
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_content_section( $html ) {
+	$content = stats_parse_content_section( $html );
+	if ( $content !== '' ) {
+		echo $content; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+}
+
+/**
+ * Legacy Stats: Print Chart Scripts
+ *
+ * @access public
+ * @param mixed $html HTML.
+ * @return void
+ */
+function stats_print_chart_scripts( $html ) {
+	if ( is_chart_scripts( $html ) ) {
+		echo $html; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	}
+}
+
+/**
+ * Legacy Stats: Test for presence of chart scripts
+ *
+ * @access public
+ * @param mixed $html Input HTML that may or may not include the chart scripts.
+ * @return bool
+ */
+function is_chart_scripts( $html ) {
+	$str = stats_content_marker();
+	$pos = strpos( $html, $str );
+	if ( $pos === false ) {
+		return true;
+	}
+	return false;
+}
+
+/**
+ * Legacy Stats: Returns content marker
+ *
+ * Used to differentiate between content and script in
+ * the returned server-generated HTML.
+ *
+ * @access public
+ * @return string
+ */
+function stats_content_marker() {
```

Comment:
I would prefer to use a class `const` for the value because it doesn't seem we would need to do anything with it.

Reply:
Sounds good. I'll add this to the Stats → Main class.

Reply:
Sorry I didn't realize the stats.php were just a bunch of functions. I moved the const out of the class back to the file because it's legacy code, and IMO there is not need to chunk to the package.